### DISCUSSION TITLE
24.4.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1719] [General]
## Releases Notes for 24.4.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/draw.io-24.4.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/draw.io-24.4.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/draw.io-universal-24.4.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/drawio-amd64-24.4.0.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/drawio-x86_64-24.4.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.0/drawio-x86_64-24.4.0.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Uses electron 28.1.0.
- #1380 , #1508 , #1638 , #1699, #1700, #1705
- Updates to [draw.io core 24.4.0](https://github.com/jgraph/drawio/blob/v24.4.0/ChangeLog). All changes from 24.2.6 and 24.4.0 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v24.4.0'>24.4.0</a>.</em>

**Comment by Ericonaldo** [comment]
There are bugs for exporting pdfs with selected shapes only in this version, which only export pdfs in fixed pages.

**Reply by alderg** [reply]
Try check "Selection Only" and "Crop" to get the selection on a single page:

<img width="371" alt="Screenshot 2024-05-14 at 06 21 24" src="https://github.com/jgraph/drawio-desktop/assets/921200/97390c3a-a425-4a95-b2b5-8368bfa333b7">

If this does not solve the problem, please open a ticket and describe the problem in more detail.

**Comment by Ericonaldo** [comment]
Thanks, it works. I missed the option as it seems like an improper translation for the Chinese language ('导出单页' means import the single page).

**Reply by alderg** [reply]
Would '裁剪' be a better translation for crop in this context?

**Reply by Ericonaldo** [reply]
Yes it is a better translation.

**Comment by Joepatoon** [comment]
Hi, i'm using mac os 10.14.6 and i can't use draw.io, i got an item saying the app needs 10.15 or further. Is there an installer for older versions of mac?
regards

**Comment by natrixiva** [comment]
Starting from this version Collapse / Expand of container doesn't work anymore. Checked with 24.2.5 version: it works.

**Reply by davidjgraph** [reply]
Works fine for me.

**Reply by natrixiva** [reply]
Don't know what's wrong in my case. See attached screen videos. 
![draw io_5d0e46FDx1](https://github.com/jgraph/drawio-desktop/assets/50548253/ec2290b9-ea8f-4ca9-a3ba-d3a864d0f6ca)
![draw io_R4jgvdvzog](https://github.com/jgraph/drawio-desktop/assets/50548253/b99fe291-1e24-4602-9d24-405dbc54e2e9)

**Reply by davidjgraph** [reply]
could you attach the XML of the diagram zipped please?

**Reply by natrixiva** [reply]
Attached both xml and drawio. Two container samples are on the tab "Pagina 16".
[Collapse-expand_error_drawio.zip](https://github.com/user-attachments/files/15821891/Collapse-expand_error_drawio.zip)
[Collapse-expand_error_xml.zip](https://github.com/user-attachments/files/15821892/Collapse-expand_error_xml.zip)

**Reply by alderg** [reply]
Check if Collapse/Expand is enabled under Extras:

<img width="303" alt="Screenshot 2024-06-14 at 10 53 12" src="https://github.com/jgraph/drawio-desktop/assets/921200/8b349ffa-c32a-42e2-beb8-beee51666bd9">

**Reply by natrixiva** [reply]
The first thing that I did. Did you check it with my file?

**Reply by davidjgraph** [reply]
We checked with the file, expand and collapse work for us.

**Reply by natrixiva** [reply]
I tried with a different PC and it works. Apparently starting with version 24.4.0 some kind of shared resource is used, which on my PC has some kind of difference.

**Reply by alderg** [reply]
Are you using a configuration (via Extras > Configuration) on the affected PC?
